platform/posix/transport/src/openssl_posix.c

```diff
@@ -74,8 +74,8 @@
  *
  * @return 1 on success; -1, 0 on failure;
  */
-static int setRootCa( SSL_CTX * pSslContext,
-                      const char * pRootCaPath );
+static int32_t setRootCa( const SSL_CTX * pSslContext,
```

Comment:
Why has const been set for this setter function but not for other setter functions for private key and client certificate?
Isn't the context expected to be modified when setting the root CA?

Reply:
Misra tagged only this as the the openssl functions called within this function expects a pointer to a const ssl context, but that is not the same with other 2 functions.